server/src/auth/google/google.controller.ts

```diff
@@ -1,4 +1,14 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Post, Request } from '@nestjs/common';
+import { AuthService } from '../auth.service';
 
 @Controller('google')
-export class GoogleController {}
+export class GoogleController {
+  constructor(private readonly authService: AuthService) {}
+
+  // login
+  @Post('/auth/login')
+  async login(@Request() req) {
+    return this.authService.loginGoogle(req.token);
```

Comment:
If you make the change I mentioned above, you can call it "login(req.body.token, AuthType.GOOGLE)." Also, please use "@Body()" instead of "@Request()" and change "@Post('...')" to "@Post()" because "@Controller('google')" already defines this route as "/google" for this controller.